Service/ZGWService.php

```diff
@@ -189,6 +210,14 @@ public function inhoudHandler(array $data, array $configuration): array
             $objectEntity instanceof ObjectEntity &&
             $objectEntity->getEntity()->getId()->toString() == $configuration['enkelvoudigInformatieObjectEntityId']
         ) {
+            if($objectEntity->getLock() !== null
+                && key_exists('lock', $this->data['body'])
+                && $objectEntity->getLock() !== $this->data['body']['lock']
+                && ($this->data['method'] === 'PUT' || $this->data['method'] === 'PATCH')
```

Comment:
Klopt het dat al deze 4 checks true moeten zijn voordat de lock invalid is of moet er || ipv &&